syncano/connection.py

```diff
@@ -290,6 +290,8 @@ def get_response_content(self, url, response):
 
         # Validation error
         if is_client_error(response.status_code):
+            if response.status_code == 400 and 'Revision mismatch.' in content.get('non_field_errors', [''])[0]:
```

Comment:
As we agreed: https://github.com/Syncano/syncano-platform/pull/823
So it should be enough to check if 'expected_revision' in content